### DISCUSSION TITLE
Add schema import feature to Schema Config editor

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5576] [Feature Requests]
Like the one in Specify 6 (but using JSON instead of XML like sp6)

**Comment by grantfitzsimmons** [comment]
Would be invaluable.

**Comment by grantfitzsimmons** [comment]
This was requested by Corinna at CSIRO today. This would be very useful.

**Comment by grantfitzsimmons** [comment]
Requested by Matthew Cruz at UMich again as well!

**Comment by grantfitzsimmons** [comment]
Necessary for Geo schema!

**Comment by grantfitzsimmons** [comment]
You can export it now, but can't import it.

**Comment by grantfitzsimmons** [comment]
Requirements have been formalized: https://github.com/specify/specify7/issues/6155
